Tests/Fixtures/TestProject/project.yml

```diff
@@ -116,6 +116,7 @@ targets:
         - App_iOS_UITests
       gatherCoverageData: true
       disableMainThreadChecker: true
+      stopOnEveryMainThreadCheckerIssue: false
```

Comment:
Can we set this to true to generate the diff in the checked in fixture 👍

Reply:
Done

Reply:
My understanding is that the default should be false because it shouldn't stop on every main thread checker issue. Am I right?